components/datetime/Cargo.toml

```diff
@@ -91,16 +91,6 @@ datagen = [
     "std",
     ]
 experimental = ["dep:litemap"]
-bench = ["serde"]
```

Comment:
Thought: The old way, you could use the Cargo features mechanism to enable serde when bench was enabled. You now lose the ability to do that.